poc/pkg/templates/mise-configMap.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+data:
+  appsettings.json: |
+    {
+      "Version": "1",
+      "AzureAd": {
+        "Instance": "https://login.microsoftonline.com",
+        "ClientId": "704c7dfd-4ed1-4732-95c0-04a44b0895a0",
+        "TenantId": "72f988bf-86f1-41af-91ab-2d7cd011db47",
+        "InboundPolicies": 
+        [
+          {
+            "Label": "jonachang-policy",
+            "AuthenticationSchemes": [ "Bearer"],
+            "ValidAudiences" : [ 
+                "https://management.azure.com/"
```

Comment:
we shouldn't use this

Reply:
which audience should i use? just the clientid for poc purpose?